### DISCUSSION TITLE
Where can I get the full documentation of the datacube configurations?

### DISCUSSION THREAD

**Discussion by lisahligono** [ZGIS/semantique#23] [Q&A]
I would like to ask if it's possible to get the full documentation of the opendatacube and geotiffarchive configuration

**Comment by luukvdmeer** [comment]
What do you mean by "full documentation"? All initialization parameters, methods and properties of the respective classes can be found here:

https://zgis.github.io/semantique/_generated/semantique.datacube.Opendatacube.html
https://zgis.github.io/semantique/_generated/semantique.datacube.GeotiffArchive.html

**Reply by lisahligono** [reply]
Thank you for the links, I was using this link: https://zgis.github.io/semantique/_notebooks/datacube.html#The-Geotiffarchive-configuration
and it stated 'to be added' under the GeoTIFFArchive configurations. I guess I was using the wrong documentation

**Reply by luukvdmeer** [reply]
These notebooks are updated now
